src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const longTermDiscount = 50;
```

Comment:
2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

please rename your constants like in the GOOD EXAMPLE